testing/environments/snapshot.yml

```diff
@@ -24,7 +24,7 @@ services:
     - "script.context.template.cache_max_size=2000"
 
   kibana:
```

Comment:
You're probably missing updates in `kibana.config.yml`: https://github.com/elastic/elastic-package/blob/master/internal/install/static_kibana_config_yml.go

Reply:
Tried to make updates to `kibana.config.yml` in 740a5ba.